Sources/Build/misc.swift

```diff
@@ -50,13 +55,9 @@ extension ClangModule {
         
         let includeDir = path
         
-        //Generate empty module map if include dir is not present
+        //Warn and return if no include directory
         guard includeDir.isDirectory else {
-            print("warning: No include directory, generating empty module map")
-            try POSIX.mkdir(includeDir)
-            try fopen(moduleMapPath, mode: .Write) { fp in
-                try fputs("\n", fp)
-            }
+            print("warning: No include directory found, a library can not be imported without any public headers.")
```

Comment:
Unrelated, but we should start to get a proper diagnostics infrastructure instead of just embedding print() statements...